sql/stored-proc/maintain-current.inc.php

```diff
@@ -2387,19 +2387,7 @@
 sql(
     "CREATE TRIGGER `userBeforeDelete` BEFORE DELETE ON `user`
      FOR EACH ROW BEGIN
-        IF IFNULL(@allowdelete,0)=0 THEN
-            CALL error_set_allowdelete_to_delete_users();
```

Comment:
This procedure call is intentional. It raises an SQL error if @allowdelete is not set, to get the developer's attention to the following comments. 

Please don't delete this code, or implement another way to throw some error or exception here which points the developer to an explanation on deleting user records.

Reply:
Btw, even if the procedure call would have been obsolete, your change would have removed the complete IF condition which prevents deleting a user if @allowdelete is not set!